### ISSUE TITLE
Running CLI give "UNKNOWN_FORMAT" errors

### ISSUE THREAD

**Issue by xied75** [zaggino/z-schema#121]
Dear all,

I was using z-schema from command line against a json schema file for https://github.com/jdorn/json-editor

I have a block like this
```
    "password": {
        "type": "string",
        "title": "Password",
        "format": "password"
    },
```

Which gives me the following error:

```
    {
        "code": "UNKNOWN_FORMAT",
        "params": [
            "password"
        ],
        "message": "There is no validation function for format 'password'",
        "path": "#/properties/password"
    },
```

According to the spec http://json-schema.org/latest/json-schema-validation.html#anchor104
_A format attribute can generally only validate a given set of instance types. If the type of the instance to validate is not in this set, validation for this format attribute and instance SHOULD succeed._

**7.2.  Implementation requirements**
_they SHOULD offer an option to disable validation for this keyword._

Therefore I wonder if z-schema should either give an option as suggested, or instead of an error, change it to a warning?

Best regards,

Dong

**Comment by simon-p-r** [comment]
Have you registered the password format? One way to check is call the getRegisteredFormats method which will return an array of strings for each format that is registered.

**Comment by xied75** [comment]
@simon-p-r I understand there are tons of method I can call if I'm writing code; unfortunately I'm in command line. Besides I'm trying to point out what the SPEC is saying which I believe tools should normally follow. :)

**Comment by simon-p-r** [comment]
@xied75  just do some preprocessing on schemas with Z-schema and then call them from command line.

**Comment by zaggino** [comment]
PR for `ignoreUnknownFormats` option is welcome

**Comment by whitlockjc** [comment]
I think we should have a flag to ignore unknown formats as well since the JSON Schema specification says that the handling of unknown formats could be ignored but is up to the implementation: http://json-schema.org/latest/json-schema-validation.html#anchor106  This just bit me in https://github.com/swagger-api/swagger-editor/issues/612

**Comment by mohsen1** [comment]
That's a long dependency chain! 
`z-schema->sway->sway-worker->swagger-editor`

**Comment by zaggino** [comment]
published as `3.13.0`
